spec/controllers/batch_controller_spec.rb

```diff
@@ -22,16 +22,25 @@
       end
     end
 
-    describe "when submiting files on behalf of another user" do
-      let(:somebody_else_file) do 
-        f = GenericFile.create(title: ['Original Title']) { |gf| gf.apply_depositor_metadata('current_user') }
-        f.on_behalf_of = 'somebody@else.org'
+    describe "when submiting works on behalf of another user" do
+      let(:somebody_else_work) do 
+        w = GenericWork.create(title: ['Original Title']) { |gw| gw.apply_depositor_metadata('current_user') }
```

Comment:
@grosscol You can save only once if you use a .tap here or add you on_behalf_of = 'somebody@else.org' in the block above. For example: https://github.com/projecthydra/sufia/blob/master/spec/models/ability_spec.rb#L44